reference/track-request.mdx

```diff
@@ -131,4 +129,139 @@ data_payload = {
 # Making the request
 response = requests.post("https://api.promptlayer.com/rest/track-request", json=data_payload)
 print(response.json())
-```
\ No newline at end of file
+```
+
+## Logging Google Requests
```

Comment:
Can you add a title like this to the codeblock above for compeltions?

Reply:
Sure